src/.env.sample

```diff
@@ -180,5 +180,8 @@ OBSERVATION_DEEP_LINK_REGEX= "^https:\/\/dev\.elevate-ml\.shikshalokam\.org\/vie
 # Mongo DB
 MONGODB_URL=mongodb://localhost:27017/elevate-project
 
+# detault data manager roles 
+DEFAULT_DATA_MANAGERS="program_manager,program_designer"
+
```

Comment:
@adithyadinesh0412 please update the postman collection, no need to change entire collection please add only this api